devfiles/python-django/devfile.yaml

```diff
@@ -7,7 +6,8 @@ projects:
     name: django-realworld-example-app
     source:
       type: git
-      location: "https://github.com/che-samples/django-realworld-example-app"
+      location: 'https://github.com/che-samples/django-realworld-example-app'
+      commitId: 0d425480c1b28d6063c255e7c4fa869eb1709d85
```

Comment:
This might need to change once the commit has been merged, I'm not sure if GH respects commit ID's the way it ought to.